packages/react-native-expo/src/schematics/application/application.impl.ts

```diff
@@ -77,7 +77,7 @@ function createApplicationFiles(options: NormalizedSchema): Rule {
 }
 
 function updateNxJson(options: NormalizedSchema): Rule {
-  return updateJsonInTree<NxJson>('nx.json', (json) => {
+  return updateJsonInTree<NxJson>('workspace.json', (json) => {
```

Comment:
First @tk-o thanks for the contribution 💜
I think this is a breaking change. Maybe the setup of a migration path is needed.
Not sure if this will be easy to do or not.
I'm also rewriting a lot of code to support Nx12, and I will merge this when that work is done.